cli/ballerina-cli/src/main/resources/cli-help/ballerina-tool-search.help

```diff
@@ -16,19 +16,19 @@ DESCRIPTION
 
 EXAMPLES
        Search by tool id.
-           $ bal tool search openapi
+           $ bal tool search health
 
        Search by package id.
-           $ bal tool search openapi
-           $ bal tool search package:openapi
+           $ bal tool search health
```

Comment:
We don't need to allow searching by org and package name. Tools are only supposed to be searched by the id. Let's remove these examples and keep only searching by id.